forest/sycamore/functions.py

```diff
@@ -1,47 +1,50 @@
 import datetime
-import glob
 import json
 import logging
 import os
-from typing import List
 
+import glob
 import numpy as np
 import pandas as pd
-import pytz
+
+from typing import Optional
 
 # Explore use of logging function (TO DO: Read wiki)
 logger = logging.getLogger(__name__)
 
-
 # Modified from legacy beiwetools code
-def read_json(study_dir):
-    """
-    Read a json file into a dictionary.
+
+
+def read_json(study_dir: str) -> dict:
+    """Read a json file into a dictionary
+
     Args:
         study_dir (str):  study_dir to json file.
     Returns:
         dictionary (dict)
     """
-    with open(study_dir, 'r') as f:
+    with open(study_dir, "r") as f:
         dictionary = json.load(f)
     return dictionary
 
 
 # load events & question types dictionary
-# From Josh's legacy script
+# From Josh"s legacy script
 this_dir = os.path.dirname(__file__)
-events = read_json(os.path.join(this_dir, 'events.json'))
-question_type_names = read_json(os.path.join(this_dir, 'question_type_names.json'))
+events = read_json(os.path.join(this_dir, "events.json"))
+question_type_names = read_json(
+    os.path.join(this_dir, "question_type_names.json")
+)
```

Comment:
Yes, I'm not sure why they did this that way. I suppose maybe this way allows for changing question types more easily if new ones are added, but it would be just as easy to update this dict as it would be to update the json file with new types.